Operators/Types/user/Dom/util/GetTextureSizeSquare.cs

```diff
@@ -0,0 +1,26 @@
+using SharpDX.Direct3D11;
+using SharpDX;
+using T3.Core.Operator;
+using T3.Core.Operator.Attributes;
+using T3.Core.Operator.Slots;
+
+namespace T3.Operators.Types.Id_4bee2cb2_effe_4129_8d0e_183374159459
+{
+    public class GetTextureSizeSquare : Instance<GetTextureSizeSquare>
```

Comment:
yes it's very specific, though since making this I've been adapting the RBush repository for spatial indexing at work (lidar sensing etc etc) and they have this wonderful [Envelope](https://github.com/viceroypenguin/RBush/blob/master/RBush/Envelope.cs) class which may provide some great inspo for how we can handle `Size` and `Vector` intersections, extensions, etc in the future